core/biogeophys/cable_canopy.F90

```diff
@@ -1666,6 +1666,9 @@ SUBROUTINE dryLeaf( dels, rad, air, met, &
     deltlfy = abs_deltlf
     k = 0
 
+    !CLN as yet uninitialised
```

Comment:
We need to keep this

Reply:
Do you mean L1669?

Reply:
line 1670 - the initialisation of cx2(:)

Reply:
It will be in your branch after the merge.